command/server/seal/server_seal_transit.go

```diff
@@ -3,22 +3,28 @@ package seal
 import (
 	"github.com/hashicorp/errwrap"
 	log "github.com/hashicorp/go-hclog"
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/hashicorp/go-kms-wrapping/wrappers/transit"
 	"github.com/hashicorp/vault/command/server"
 	"github.com/hashicorp/vault/sdk/logical"
 	"github.com/hashicorp/vault/vault"
-	"github.com/hashicorp/vault/vault/seal/transit"
+	"github.com/hashicorp/vault/vault/seal"
 )
 
 func configureTransitSeal(configSeal *server.Seal, infoKeys *[]string, info *map[string]string, logger log.Logger, inseal vault.Seal) (vault.Seal, error) {
-	transitSeal := transit.NewSeal(logger)
+	transitSeal := transit.NewWrapper(&wrapping.WrapperOptions{
+		Logger: logger.ResetNamed("seal-transit"),
```

Comment:
Isn't this redundant?  It looks like the caller is already using `logger.Named(sealType)`.

Reply:
This is what the code in vault/seal/transit was already doing so I kept it this way for backwards compat, but moved it out here rather than leak that into the library.